src/engraving/compat/engravingcompat.cpp

```diff
@@ -37,6 +37,10 @@ using namespace mu::engraving;
 namespace mu::engraving::compat {
 void EngravingCompat::doPreLayoutCompatIfNeeded(MasterScore* score)
 {
+    if (score->mscVersion() >= 450) {
```

Comment:
Somehow it feels unexpected to see "compat" code that only applies to the latest version... are you sure that this method does the right thing for the right version?